Description: adjust whitespace as required for recent autotools
 Recent releases of autotools (possibly only autoreconf
 apparently only recognizes the macro IT_PROG_INTLTOOL
 when listed as the first word on a line.
 Otherwise autoreconf fails with the following error message:
 .
 autoreconf: running: intltoolize --copy --force
 ERROR: 'IT_PROG_INTLTOOL' must appear in configure.ac for intltool to work.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2021-10-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -587,11 +587,11 @@
 AC_DEFINE_UNQUOTED(DBUS_SERVICES_DIR, "$DBUS_SERVICES_DIR", [Location of D-Bus services directory])
 
 if test "$enable_gui" != "no" || test "$enable_dbus_service" != "no"; then
-    IT_PROG_INTLTOOL([0.37.1])
-    GETTEXT_PACKAGE=syncevolution
-    AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [The gettext package name])
-    AM_GLIB_GNU_GETTEXT
-    SYNCEVOLUTION_LOCALEDIR=[${datadir}/locale]
+IT_PROG_INTLTOOL([0.37.1])
+GETTEXT_PACKAGE=syncevolution
+AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [The gettext package name])
+AM_GLIB_GNU_GETTEXT
+SYNCEVOLUTION_LOCALEDIR=[${datadir}/locale]
 fi
 
 # decide which sync-ui(s) we are building:
